MetaMask/metamask-extension
# feat: use Snaps directory to list Account Management Snaps

## **Description**

This PR removes the embedded Account Management Snaps listing and replaces it with [Snaps registry](https://snaps.metamask.io). 

> ⚠️ **Warning:** As a side effect, this PR also removes the modal explaning what account Snaps are, and the link to the feedback form.

**This PR is missing:**

- [x] The final production URL
- [x] The final staging URL

> ✏️ **Note:** I updated this PR with the final URLs, but they are not deployed yet.

**Depends on (for production):**

- <https://github.com/MetaMask/snaps-directory/pull/164>

## **Related issues**

Fixes: <https://github.com/MetaMask/accounts-planning/issues/145>

## **Manual testing steps**

Repeat the following steps for both the **expanded** and **compact** views:

1. Install **Flask**
2. Enable the **Add account Snaps** button:
   1. Go to **Settings** > **Experimental** > **Account Snaps**
   2. Click on **Enable "Add account Snap (Beta)"**
3. Go to the account picker
4. Click on **Add account or hardware wallet**
5. Click on **Add account Snap**
6. A new tab should open, the URL should be <https://snaps.metamask.io/account-management>

> ⚠️ **Warning:** This cannot be tested on MetaMask stable until the following PR gets merged:
>
> - <https://github.com/MetaMask/metamask-extension/pull/22166>.

## **Screenshots/Recordings**



### **Before**

<img width="1602" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/68558152/be484d72-490f-4323-865f-6dce20cdd4e1">

### **After**

> ✏️ **Note:** For demonstration purposes, the following screenshot shows the **Interoperability** category instead of **Account Management**.

<img width="1288" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/68558152/f4f417f4-e5c0-4ca7-aa09-775af82031f5">

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
